<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RDP</class>
 <widget class="QWidget" name="RDP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>364</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="load_waypoints_button">
       <property name="text">
        <string>Load waypoints</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="simplify_button">
       <property name="text">
        <string>Simplify waypoints</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_waypoints_button">
       <property name="text">
        <string>Save waypoints</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="epsilon_label">
         <property name="text">
          <string>Epsilon:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="epsilon_line_edit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>188</width>
         <height>188</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
